fiftyone/core/storage.py

```diff
@@ -281,9 +281,9 @@ def join(a, *p):
     return os.path.join(a, *p)
 
 
-def resolve(path):
-    """Resolves path to absolute, resolving symlinks and relative path
-        indicators such as `.` and `..`.
+def realpath(path):
```

Comment:
Just noting this is a public API change

Reply:
Yes true. My thinking was that this was only introduced in `fiftyone==0.24.0` and the spirit of the `fiftyone.core.storage` module is to exactly match the `os` package's naming convention for the methods that it wraps, so this felt consistent.